director/settings.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, home_path=None, config_path=None):
         self.FLOWER_URL = env.str("DIRECTOR_FLOWER_URL", "http://127.0.0.1:5555")
         self.WORKFLOWS_PER_PAGE = env.int("WORKFLOWS_PER_PAGE", 1000)
 
+        # Authentication
+        self.AUTH_ENABLED = env.bool("DIRECTOR_AUTH_ENABLED", False)
```

Comment:
Can you add this configuration in the template of `director init` command ? For example :

```
# Authentication
DIRECTOR_AUTH_ENABLED = false
```

In this file : https://github.com/ovh/celery-director/blob/master/director/commands/init.py#L6